### PR TITLE
Docker: change ivre/db on mongo

### LINKED ISSUE #1525: [Ivre DB] CPU high usage
### IVRE Version
```
$ ivre version
IVRE - Network recon framework
Copyright 2011 - 2023 Pierre LALET <pierre@droids-corp.org>
Version 0.9.20.dev81-dockerhub

Python 3.9.2 (default, Feb 28 2021, 17:03:44)
[GCC 10.2.1 20210110]

Linux 4637a07134ad 5.15.49-linuxkit #1 SMP PREEMPT Tue Sep 13 07:51:32 UTC 2022 x86_64

Dependencies:
    MySQLdb: *missing*
    OpenSSL: 20.0.1
    PIL: 8.1.2
    bottle: 0.12.19
    cryptography: 3.3.2
    dbus: *missing*
    krbV: *missing*
    matplotlib: *missing*
    psycopg2: *missing*
    pycurl: *missing*
    pymongo: 3.11.0
    sqlalchemy: *missing*
    tinydb: *missing*
```

### Summary

Hey! 
I installed Ivre with this [docker-compose ](https://github.com/ivre/ivre/blob/master/docker/docker-compose.yml), but something wrong with DB. CPU usage 100% and I can't run any command because it's connection timeout

### How to reproduce

1) clone https://github.com/ivre/ivre
2) cd docker
3) docker compose up
4) docker attach ivreclient
5) ivre ipinfo --init

### More

Output:
```
Traceback (most recent call last):
  File "/usr/local/bin/ivre", line 102, in <module>
    main()
  File "/usr/local/bin/ivre", line 70, in main
    tools.get_command(next(iter(possible_commands)))()
  File "/usr/local/lib/python3.9/dist-packages/ivre/tools/ipinfo.py", line 318, in main
    dbase.init()
  File "/usr/local/lib/python3.9/dist-packages/ivre/db/mongo.py", line 298, in init
    self.db[colname].drop()
  File "/usr/lib/python3/dist-packages/pymongo/collection.py", line 1127, in drop
    dbo.drop_collection(self.__name, session=session)
  File "/usr/lib/python3/dist-packages/pymongo/database.py", line 921, in drop_collection
    with self.__client._socket_for_writes(session) as sock_info:
  File "/usr/lib/python3/dist-packages/pymongo/mongo_client.py", line 1293, in _socket_for_writes
    server = self._select_server(writable_server_selector, session)
  File "/usr/lib/python3/dist-packages/pymongo/mongo_client.py", line 1278, in _select_server
    server = topology.select_server(server_selector)
  File "/usr/lib/python3/dist-packages/pymongo/topology.py", line 241, in select_server
    return random.choice(self.select_servers(selector,
  File "/usr/lib/python3/dist-packages/pymongo/topology.py", line 199, in select_servers
    server_descriptions = self._select_servers_loop(
  File "/usr/lib/python3/dist-packages/pymongo/topology.py", line 215, in _select_servers_loop
    raise ServerSelectionTimeoutError(
pymongo.errors.ServerSelectionTimeoutError: ivredb:27017: [Errno 111] Connection refused, Timeout: 30s, Topology Description: <TopologyDescription id: 647dc0211cd52d962f330202, topology_type: Single, servers: [<ServerDescription ('ivredb', 27017) server_type: Unknown, rtt: None, error=AutoReconnect('ivredb:27017: [Errno 111] Connection refused')>]>
```

docker stats:
```
CONTAINER ID   NAME         CPU %     MEM USAGE / LIMIT     MEM %     NET I/O           BLOCK I/O        PIDS
7665faf6be0c   ivredoku     0.02%     26.91MiB / 11.68GiB   0.22%     1.34kB / 0B       0B / 4.1kB       8
a7dfb8f12563   ivredb       100.49%   16.38MiB / 11.68GiB   0.14%     10.4kB / 6.8kB    0B / 0B          4
4637a07134ad   ivreclient   0.00%     18.11MiB / 11.68GiB   0.15%     7.92kB / 9.24kB   10.7MB / 4.1kB   2
07ea27b2b99a   ivreuwsgi    0.02%     48.14MiB / 11.68GiB   0.40%     1.12kB / 0B       0B / 0B          4
7db0a8d39a27   ivreweb      0.00%     23.75MiB / 11.68GiB   0.20%     1.12kB / 0B       0B / 4.1kB       14
```

image 1984
image 1985

```uname -a
Darwin bairovp.local 22.5.0 Darwin Kernel Version 22.5.0: Mon Apr 24 20:52:24 PDT 2023; root:xnu-8796.121.2~5/RELEASE_ARM64_T6000 arm64
```

### PR DESCRIPTION
**Pull request by Amerousful**
Fixes #1525

### CODE REVIEW

**Review by p-l- — CHANGES_REQUESTED** [review]
Hey, thanks for this. I think you should also:
- remove the `docker/db` folder
- update `docker/Vagrantfile`
- remove the `/var/log/mongodb` mount point (and check if the logs go to std{out,err} in the official image, or if it is safe to just remove it)
- update the `/var/lib/mongodb` (I don't think the official container uses the same directory for its data).

WDYT?

**Comment by Amerousful** [comment]
I changed `/var/lib/mongodb` on `/data/db
image 2014
`

File: docker/Vagrantfile [hunk 1]
```diff
@@ -48,11 +48,10 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # The DB server
   config.vm.define 'ivredb' do |db|
     db.vm.provider "docker" do |docker|
-      docker.image = 'ivre/db'
+      docker.image = 'mongo'
       docker.name = 'ivredb'
     end
     db.vm.synced_folder "var_lib_mongodb", "/var/lib/mongodb"
```
Comment (p-l-) [inline docker/Vagrantfile:54]: We're almost there: I think you have forgotten to update the container path here!

**Review by p-l- — CHANGES_REQUESTED** [review]
Thanks a lot for updating this PR! You just need to update the Vagrantfile.
Also, can you remove `db` from `pkg/builddockers` (twice)?

File: docker/Vagrantfile (hunk 1 above)
Comment (Amerousful) [reply docker/Vagrantfile:54]: Oh, yep. Thanks for the patient review)
One sec

**Comment by Amerousful** [comment]
@p-l- You're a patient reviewer, thank you)
I hope I fixed it properly all remarks

**Comment by p-l-** [comment]
Hey, in `pkg/builddockers`, please **only** remove the `db` (including in the comment please), leave all the rest as it is!

**Review by p-l- — APPROVED** [review]

**Comment by Amerousful** [comment]
Hm, something wrong with `/tmp/webserver.log`...

**Comment by p-l-** [comment]
No, you need to update setup.py and remove the line `("share/ivre/docker/db", ["docker/db/Dockerfile"]),`.

Also, can you apply the following patch:
```diff
diff --git a/doc/install/docker.rst b/doc/install/docker.rst
index a3e20789..764f2525 100644
--- a/doc/install/docker.rst
+++ b/doc/install/docker.rst
@@ -55,9 +55,9 @@ data):
 
 ::
 
-   $ mkdir -m 1777 var_{lib,log}_mongodb ivre-share dokuwiki_data
+   $ mkdir -m 1777 var_lib_mongodb ivre-share dokuwiki_data
      # For people using SELinux enforced, you need to run
-   $ sudo chcon -Rt svirt_sandbox_file_t var_{lib,log}_mongodb ivre-share dokuwiki_data
+   $ sudo chcon -Rt svirt_sandbox_file_t var_lib_mongodb ivre-share dokuwiki_data
 
    $ cp [path to ivre source]/docker/Vagrantfile .
    $ vagrant up --no-parallel
@@ -79,7 +79,7 @@ that, from the ``docker/`` directory, run:
 
    $ docker pull debian:stable
    $ docker pull debian:buster
-   $ for img in base client agent db web web-doku web-uwsgi ; do
+   $ for img in base client agent web web-doku web-uwsgi ; do
    > docker build -t "ivre/$img" "$img"
    > done
 
diff --git a/docker/Vagrantfile b/docker/Vagrantfile
index 5f4238b5..398656bd 100644
--- a/docker/Vagrantfile
+++ b/docker/Vagrantfile
@@ -19,14 +19,14 @@
 
 
 #
-# Create the directories to store the logs and data:
+# Create the directory to store the data:
 #
-# $ mkdir -m 1777 var_{lib,log}_mongodb ivre-share dokuwiki_data
+# $ mkdir -m 1777 var_lib_mongodb ivre-share dokuwiki_data
 #
 #
 # If you have SELinux enforced, run:
 #
-# $ sudo chcon -Rt svirt_sandbox_file_t var_{lib,log}_mongodb ivre-share dokuwiki_data
+# $ sudo chcon -Rt svirt_sandbox_file_t var_lib_mongodb ivre-share dokuwiki_data
 #
 #
 # Then start the containers with:
```

**Review by p-l- — CHANGES_REQUESTED** [review]
See my comment: you need to fix `setup.py`; also, please apply the other changes provided to update the documentation accordingly!
Thank you!

**Comment by Amerousful** [comment]
Got it. Thanks for review

**Comment by p-l-** [comment]
Thank you!

**Pull request merged**